sdk/communication/communication-common/package.json

```diff
@@ -80,6 +80,7 @@
     "@types/mocha": "^10.0.0",
     "@types/node": "^16.0.0",
     "@types/sinon": "^10.0.0",
+    "c8": "^8.0.1",
```

Comment:
Hi @ozgurcivi-msft! I was wondering why the package was added to the common sdk :)

Reply:
@AikoBB, "nyc" is broken in latest NodeJS versions ([link](https://github.com/istanbuljs/nyc/issues/1530)), so I was trying to use c8.
And because the similar PR is done by others, I'll cancel this PR.